spec/unit/provider/package/apt_spec.rb

```diff
@@ -590,7 +650,7 @@ def ubuntu1404downgrade_stubs
       end
     end
 
-    describe "#action_install" do
+    describe "#action_upgrade" do
```

Comment:
all of these tests call action upgrade, not action install, so I fixed the name.